src/PhpSpreadsheet/Worksheet/Table/Column.php

```diff
@@ -0,0 +1,202 @@
+<?php
+
+namespace PhpOffice\PhpSpreadsheet\Worksheet\Table;
+
+use PhpOffice\PhpSpreadsheet\Worksheet\Table;
+
+class Column
+{
+    /**
```

Comment:
Do you have any plan to add column style id and calculated columns?
e.g.
```xml
<tableColumn name="Unit Price" xr3:uid="{CF759E84-F4C3-4E91-8817-327D2F6A5BD9}" id="4" dataDxfId="1"/>
```
With the `dataDxfId` here pointing to a money style mask

and 

```xml
<tableColumn name="Value" xr3:uid="{A067D829-D65F-4AA3-996D-6F464FDADAD0}" id="5" dataDxfId="0">
    <calculatedColumnFormula>Table2[[#This Row],[Quantity]]*Table2[[#This Row],[Unit Price]]</calculatedColumnFormula>
</tableColumn>
```
Which allow the formula and style to be applied automatically for any new rows added to the table range

Reply:
Yes. I have a plan to add both column style id and calculated columns.

Reply:
Tried to add column style but I failed. Because I couldn't find a way to get the id (zero-based index) of the style in `dxfs` of `styles.xml`.

ColumnFormula is added successfully. Note that only the new rows make use of the column formula, for existing rows the formula needs to be added to each cell in the column.